tests/installation/start_install.pm

```diff
@@ -82,7 +83,21 @@ sub run {
         assert_screen("inst-packageinstallationstarted", $started_timeout);
     }
     else {
-        sleep 2;    # textmode is sometimes pressing alt-i too early
+        if (check_var('DESKTOP', 'kde')) {
+            send_key_until_needlematch 'packages-section-selected', 'tab';
+            send_key 'ret';
+            send_key_until_needlematch 'gnome-desk-env', 'down';
+            send_key ' ';    #deselect gnome
+            send_key_until_needlematch 'kde-desk-env', 'down';
+            send_key ' ';    #select kde
+            send_key 'alt-o';
```

Comment:
I see no synchronisation between the indidvidual actions so this will likely be unstable. Please add according screen checks, e.g. with `assert_screen` between each check. Alternatives (less recommended) would be `wait_still_screen` and alike

Reply:
Code updated.